{{ cookiecutter.project_name_dashed }}/README.md

```diff
@@ -126,6 +127,24 @@ make u
 
 ### Environment reproducibility
 
+Below mentioned are some tools and configuration that you can use to make your
```

Comment:
Please add how `asdf` and Dev Containers differ. What are the use cases for each? How do they work hand in hand?

Reply:
Yeah, DinD sucks. But that affects only Dev Containers, no `asdf`?
But yeah, given that _most_ of our services are Docker-basded, this is indeed a problem. But not to discuss here - so feel free to close this.